src/components/profile/ProfilePrivacy.js

```diff
@@ -61,9 +53,16 @@ const ProfilePrivacy = props => {
   const { screenProps, styles, theme } = props
   const [showDialog] = useDialog()
   const gdstore = GDStore.useStore()
-  const { avatar } = gdstore.get('profile')
 
+  // bordered box required data
+  const { avatar } = gdstore.get('profile')
   const avatarSource = useMemo(() => (avatar ? { uri: avatar } : unknownProfile), [avatar])
+  const faceRecordId = useMemo(() => {
+    const enrollmentIdentifier = userStorage.getFaceIdentifier()
+    return isSmallDevice
```

Comment:
i remember you've already asked me how to perform shortify with middle dots like this one
how you solved this issue on that case ? let's try to reuse that solution here too
and let's move it somewhere to the src/lib